Microsoft.Toolkit.Win32/Microsoft.Windows.Interop.WindowsXamlHost.WinForms/WindowsXamlHost.cs

```diff
@@ -92,76 +61,63 @@ public AutoSizeMode AutoSizeMode
         [Category("XAML")]
         public string InitialTypeName
         {
-            get; set;
+            get;
+            set;
         }
 
         /// <summary>
         ///    Gets or sets XAML content for XamlContentHost
         /// </summary>
         [Browsable(false)]
         [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
-        public global::Windows.UI.Xaml.UIElement XamlRoot
+        public Windows.UI.Xaml.UIElement XamlRoot
         {
-            get
-            {
-                return this.desktopWindowXamlSource.Content;
-            }
+            get => DesktopWindowXamlSource.Content;
 
             set
             {
                 if (!DesignMode)
                 {
-                    global::Windows.UI.Xaml.FrameworkElement newFrameworkElement = value as global::Windows.UI.Xaml.FrameworkElement;
-                    global::Windows.UI.Xaml.FrameworkElement oldFrameworkElement = this.desktopWindowXamlSource.Content as global::Windows.UI.Xaml.FrameworkElement;
+                    var newFrameworkElement = value as Windows.UI.Xaml.FrameworkElement;
+                    var oldFrameworkElement = DesktopWindowXamlSource.Content as Windows.UI.Xaml.FrameworkElement;
 
                     if (oldFrameworkElement != null)
                     {
-                        oldFrameworkElement.SizeChanged -= this.FrameworkElement_SizeChanged;
+                        oldFrameworkElement.SizeChanged -= FrameworkElement_SizeChanged;
```

Comment:
Method naming (`FrameworkElement_SizeChanged` -> `OnFrameworkElementSizeChanged`)?

Reply:
Yes, this name changed.